src/pudl/transform/ferc1.py

```diff
@@ -2455,6 +2455,7 @@ def process_dbf(self, raw_dbf: pd.DataFrame) -> pd.DataFrame:
         df = (
             super()
             .process_dbf(raw_dbf)
+            .pipe(self.to_numeric)
```

Comment:
I thought about creating a more generic `to_numeric()` method that could be used by any transformer and parameterized, but the fact that it needs to be applied in `transform_start()` in `fuel_ferc1` and that's the main place that it needs to be applied meant the general case really wasn't going to be applicable.  So I punted for the moment. If we come across more of this kind of issue we should probably generalize / parameterize the method.

Reply:
I think punting is great! If we run into a need to reuse this code somewhere else, I think we should consider whether to stick it in some sort of helper function or as a mostly-unused method on the parent class.

Reply:
The *idea* with this partitioning of the parameters, functions, and classes was that we could look at the parameters to get some idea of what's being done in the transforms -- at least in the cases where it's a thing we're doing repeatedly in multiple classes (where repeatedly probably means... 3+ times?) since that makes it easier to keep track of where all a particular processing step is being applied. So I would tend toward wanting to add this to as a column + table transformer. But also very open to hearing thoughts on the overall design.